spec/integration/add_to_rails_app_spec.rb

```diff
@@ -5,9 +5,10 @@
 RSpec.describe "Adding Stitches to a New Rails App", :integration do
   let(:work_dir) { Dir.mktmpdir }
   let(:rails_app_name) { "swamp-thing" }
+  let(:rails_root) { Pathname(work_dir) / rails_app_name }
 
   def run(command)
-    stdout, stderr, stat = Open3.capture3(command)
+    stdout, stderr, stat = Open3.capture3({ 'BUNDLE_GEMFILE' => rails_root.join('Gemfile').to_path }, command)
```

Comment:
It's necessary to specify `BUNDLE_GEMFILE` because this environment variable is set in the CI build matrix to the different ruby version/rails version Gemfiles, however we want to use the normal `Gemfile` within the test application.